data-collections-exercises.md

```diff
@@ -9,95 +9,96 @@ planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
 Access the second value in `planeteers`.
 
 ```rb
-# Your answer here
+# By using the name of array and the number that value like : planeteers[1]
+the answer is 'Wind'
 ```
 
 Add "Heart" to the end of `planeteers`.
 
 ```rb
-# Your answer here
+# By using the name of the array and .push , to add any value to the last of the array like planeteers.push("Heart")
 ```
 
 Remove "Captain Planet" from the array (without using a method).
 
 ```rb
-# Your answer here
+# By using  the name of the array and - [the name of element] like: planeteers - ["Captain Planet"]
 ```
 
 Combine `planeteers` with `rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]` and save the result in a variable called `heroes`.
 
 ```rb
-# Your answer here
+# By assign the new array to this : heroes = planeteers.push(rangers)
 ```
 
 Alphabetize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# TO alphabetize the contents , should use .sort to sort the words based on alphaets like : heroes.sort
 ```
 
 Randomize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# To make the array prints its conternts random, the best method is .shuffle like: heroes.shuffle
 ```
 
 #### Bonus
 
 Select a random element from `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# To select random element in array , we should write the name of array and using method sample like : heroes.sample
 ```
 
 Select all elements in `heroes` that begin with "B" using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
 
 ```rb
-# Your answer here
+# answer = heroes.select {|word| word.include?"B"} 
```

Comment:
Great way of solving this problem but using include will return any word that has "B" character in it , we want to check the first character so we can use 

```js
heroes.select{|heros| heros[0] === 'B'}
// or
heroes.select{|heros| heros.start_with? 'B'}
```